docs/source/upgrade-guide/index.md

```diff
@@ -30,6 +30,93 @@ the latest changes, and propose to you to merge the changes, so you can run it o
 
 ## Upgrading to Volto 16.x.x
 
+### `volto-slate` is now core
+
+From Volto 16.0.0-alpha.15 onwards, `volto-slate` is integrated into Volto core and enabled as the default text block.
+The previous text block `text` based on `draftJS` is now deprecated and is restricted (hidden) in Volto bootstrap.
+This is a major change and should be planned in advance before you install 16.0.0-alpha.15 or later.
+
+```{versionadded} 16.0.0-alpha.15
+`volto-slate` added to Volto core as the default text block.
+```
+
+```{deprecated} 16.0.0-alpha.15
+`text` text block based on `draftJS` is now deprecated and might be removed from core in a future major version.
+```
+
+```{note}
+Volto 16 is intended to be the long-term support (LTS) version used when Plone 6 Final is released.
+Volto 16 will ensure compatibility with all existing Volto deployments for versions XXX and won't introduce any hard breaking change.
```

Comment:
@stevepiercy The idea is that Volto 16 won't cut off any existing installation by forcing a complicated or hard migration. So, following upgrade guide one could upgrade a Volto deployment to Volto 16. How could we express that correctly?

Reply:
I think I need to know what that statement means. When I do an upgrade to Volto 16, from what versions may I upgrade with an "easy" process? Is it step-wise, i.e., 13 -> 14 -> 15 -> 16, or rip the Band-Aid off 13 -> 16?

Also LTS has no relevance to upgrades or migrations. It means that a specific version is maintained longer than other versions, e.g., Volto 17 will be the bleeding edge with all the shiny toys.

Reply:
Ah, how about "backward compatibility"? I think that is the term you seek. For example, "Volto 16 is backward compatible to version X.Y".

Reply:
Here, remaining to be compatible equals, draftJS is still in there, so when you upgrade you still have the option to stay with it. Nothing more. Maybe then, the statement is too grandiloquent? Since draftJS is in there since the beginning, we are not cutting any previous Volto version from being upgraded to 16.

Reply:
Yes, "backward compatible" is too grand for this one feature.

I think we should designate a future version of Volto where draftJS is scheduled for removal. That will remove uncertainty and nudge the reticent developer to migrate to Slate. draftJS will not be supported forever in core Volto.

As far as what is happening with the text block in the docs, let's keep it simple.

- Slate is the new default text block.
- DraftJS is deprecated.
- DraftJS will be supported until ____?
- Developers are strongly encouraged to migrate existing DraftJS sites to Slate.

Reply:
@stevepiercy what about now?

Reply:
This is much improved. Oh shit you just merged it while I was reviewing it.

Reply:
Resolved in https://github.com/plone/volto/pull/3529